src/Our.Umbraco.InnerContent/Web/UI/App_Plugins/InnerContent/js/innercontent.controllers.js

```diff
@@ -8,17 +8,15 @@ angular.module("umbraco").controller("Our.Umbraco.InnerContent.Controllers.DocTy
 
         $scope.add = function () {
             $scope.model.value.push({
-                // All stored content type aliases must be prefixed "ic" for easier recognition.
-                // For good measure we'll also prefix the tab alias "ic" 
-                icContentTypeAlias: "",
+                icContentTypeGuid: "",
                 icTabAlias: "",
                 nameTemplate: ""
             });
         }
 
         $scope.selectedDocTypeTabs = function (cfg) {
             var dt = _.find($scope.model.docTypes, function (itm) {
-                return itm.alias.toLowerCase() === cfg.icContentTypeAlias.toLowerCase();
+                return itm.guid.toLowerCase() == cfg.icContentTypeGuid.toLowerCase();
```

Comment:
@mattbrailsford Is this intentionally a double-equals?
I'm never sure whether we need a double or triple equals in JS for string comparison. I guess in this case, we know that `toLowerCase` will return a string type, so the triple-equals doesn't need to do the type checking - does that sound right?

Reply:
I've no idea, I should really look into the proper use :D